From: Ichthyostega <prg@ichthyostega.de>
Date: Mon, 16 Jan 2017 01:53:39 +0100
Subject: cmake allow build with QT5 from Debian-Jessie

CMake does not seem to be able to pick up those features
when installing the standard development packages for QT5 on Debian/Jessie

Yet build works as expected, when the following DEB packages are installed
	qtbase5-dev (>= 5.3),
	qttools5-dev-tools,
	libqt5help5,
	libqt5svg5-dev,
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6dc636f..ab9308d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -82,9 +82,9 @@ FIND_PACKAGE(Qt5Core REQUIRED)
 FIND_PACKAGE(Qt5Gui REQUIRED)
 FIND_PACKAGE(Qt5Svg REQUIRED)
 FIND_PACKAGE(Qt5Widgets REQUIRED)
-FIND_PACKAGE(Qt5LinguistTools REQUIRED)
+#FIND_PACKAGE(Qt5LinguistTools REQUIRED)
 FIND_PACKAGE(Qt5Xml REQUIRED)
-FIND_PACKAGE(Qt5Help REQUIRED)
+#FIND_PACKAGE(Qt5Help REQUIRED)
 FIND_PACKAGE(Qt5PrintSupport REQUIRED)
 
 # in the following lines all the requires include directories are added
